### PR TITLE
Add "Manual" Control status and update RHEL8 CIS Control file

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Adds a new Control Status `manual` to use in items that do not expect automation
- Add `manual` status to Manual items from RHEL8 CIS
- Update `controleval.py` script to compute the `manual` status
- Select rules and update status of `6.2.7`, `6.2.8` and `6.2.20`.

#### Rationale:

- Update RHEL8 CIS state

### CODE REVIEW

**Comment by yuumasato** [comment]
Output of `controleva.py`
```
$ python3 utils/controleval.py stats  -i cis_rhel8 -l l2_server -p rhel8
Total controls = 237
Applicable = 100.0% -- 237 / 237
Assessed = 74.68% -- 177 / 237

Automated = 68.78% -- 163 / 237
Manual = 10.13% -- 24 / 237
Supported = 0.0% -- 0 / 237
Documentation = 0.0% -- 0 / 237
Inherently Met = 0.0% -- 0 / 237
Partial = 5.91% -- 14 / 237
Missing: 1.8.1.3, 1.8.1.6, 3.4.2.2, 3.4.2.3, 3.4.3.2, 3.4.3.3, 3.4.3.4, 3.4.3.6, 3.4.3.7, 3.4.3.8, 3.4.4.1.1, 3.4.4.1.2, 3.4.4.1.4, 3.4.4.1.5, 3.4.4.2.1, 3.4.4.2.2, 3.4.4.2.4, 3.4.4.2.5, 4.1.13, 4.1.16, 4.2.1.3, 4.2.3, 5.2.15, 5.2.2, 5.2.3, 5.3.1, 5.3.2, 5.3.3, 5.4.2, 5.5.1.5, 6.2.1, 6.2.10, 6.2.12, 6.2.14, 6.2.19, 6.2.9
```

File: controls/cis_rhel8.yml [hunk 1]
```diff
@@ -2326,6 +2328,6 @@ controls:
     levels:
       - l1_server
       - l1_workstation
-    automated: no  # The rule below exists, but does not have any OVAL checks or remediations.
-    related_rules:
+    status: automated
+    rules:
       - accounts_user_interactive_home_directory_exists
```
Comment (Mab879) [inline controls/cis_rhel8.yml:2333]: This might be the source of the missing cce.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Other than than the missing CCEs, this looks good to me.

File: utils/controleval.py [hunk 2]
```diff
@@ -64,8 +65,8 @@ def calculate_stats(ctrls):
     applicablelist = ctrls - ctrllist[controls.Status.NOT_APPLICABLE]
     assessedlist = set().union(ctrllist[controls.Status.AUTOMATED]).union(ctrllist[controls.Status.SUPPORTED])\
         .union(ctrllist[controls.Status.DOCUMENTATION]).union(ctrllist[controls.Status.INHERENTLY_MET])\
-        .union(ctrllist[controls.Status.PARTIAL])
-    print("Missing:", ", ".join(sorted(c.id for c in applicablelist - assessedlist)))
+        .union(ctrllist[controls.Status.PARTIAL]).union(ctrllist[controls.Status.MANUAL])
+    print("Missing:", ", ".join(sorted(str(c.id) for c in applicablelist - assessedlist)))
```
Comment (yuumasato) [inline utils/controleval.py:69]: I forgot to mention that the change in line 69 fixes an issue when processing CIS control files:
```
python3 utils/controleval.py stats  -i cis_rhel7 -l l2_server -p rhel7
Total controls = 248
Applicable = 100.0% -- 248 / 248
Assessed = 70.97% -- 176 / 248

Automated = 63.71% -- 158 / 248
Supported = 0.0% -- 0 / 248
Documentation = 0.0% -- 0 / 248
Inherently Met = 0.0% -- 0 / 248
Partial = 7.26% -- 18 / 248
Traceback (most recent call last):
  File "/home/wsato/git/content/utils/controleval.py", line 123, in <module>
    main()
  File "/home/wsato/git/content/utils/controleval.py", line 119, in main
    subcmds[args.subcmd](controls_manager, args)
  File "/home/wsato/git/content/utils/controleval.py", line 82, in stats
    calculate_stats(ctrls)
  File "/home/wsato/git/content/utils/controleval.py", line 68, in calculate_stats
    print("Missing:", ", ".join(sorted(c.id for c in applicablelist - assessedlist)))
TypeError: '<' not supported between instances of 'float' and 'str'

```

**Comment by yuumasato** [comment]
Just resolved a conflict with the added CIS references.

**Review by Mab879 — APPROVED** [review]

**Pull request merged**